nuxeo-platform-spreadsheet-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-spreadsheet/nuxeo-spreadsheet.html

```diff
@@ -0,0 +1,8 @@
+<link rel="import" href="nuxeo-spreadsheet-button.html">
+
+<nuxeo-slot-content name="spreadsheetButton" slot="DOCUMENT_LISTING_ACTIONS">
+  <template>
+    <nuxeo-spreadsheet-button base-url="[[baseUrl]]" nx-provider="[[nxProvider]]" mode="[[displayMode]]">
```

Comment:
Not a fan of the `mode` attribute because the button only works in `table` mode anyway, but I understand the limitations of filtering here without support for expressions.....